### ISSUE TITLE
[@nx-tools/container] Failed github action - missing github token

### ISSUE THREAD

**Issue by hardy925** [gperdomor/nx-tools#650]
Following the guide https://github.com/gperdomor/nx-tools/blob/main/packages/nx-container/docs/ci/github-actions.md#example-with-docker I end up with an error of "missing github token" when removing the tmp build folder.

<img width="1058" alt="Screen Shot 2022-11-23 at 15 10 04" src="https://user-images.githubusercontent.com/27963753/203637611-6192dcbd-43f7-47b4-a752-5b70fe403fac.png">

Am I missing something?

**Comment by hardy925** [comment]
For those that might find this issue I figured it our from reading the [Inputs](https://github.com/gperdomor/nx-tools/blob/main/packages/nx-container/docs/inputs.md) document linked at the bottom of the [@nx-tools/container/README.md](https://github.com/gperdomor/nx-tools/blob/main/packages/nx-container/README.md). In the "Inputs" file they note you can pass in variables

<img width="1100" alt="Screen Shot 2022-11-24 at 16 10 32" src="https://user-images.githubusercontent.com/27963753/203863532-0ef04583-9abe-48b8-a577-49a4614e9463.png">

However I couldn't get that to work (where do I put the `with` when my `step` doesn't use `uses`).

I ended up finding a test file that allowed [`process.env['GITHUB_TOKEN']`](https://github.com/gperdomor/nx-tools/blob/2b078709cc3f930915a73163ba575c54d7f81c92/packages/container-metadata/src/lib/meta.spec.ts#L58). With a wild guess I tired it using the note from the "Inputs" file to prefix it with `INPUT_` and it worked!


TL;DR:

```yml
      - name: Run nx affected container
        run: INPUT_GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} npx nx affected --base=$NX_BASE --head=$NX_HEAD --target=container
```

**Comment by hardy925** [comment]
resolved via https://github.com/gperdomor/nx-tools/pull/651
